.gitignore

```diff
@@ -46,8 +46,6 @@ my/
 # Files generated by setup.py
 dist/
 build/
-
-# Files generated by setupegg.py
 sympy.egg-info/
```

Comment:
This directory will still be generated by `python setup.py sdist` or `python -m build --sdist`.